conf/messages

```diff
@@ -530,6 +530,30 @@ label.next_page_forgotten_userid=On the next page, go to ‘Problems signing in
 
 label.home_page.utr=Your Self Assessment Unique Taxpayer Reference (UTR) is
 
+#*************************************************************
+# Form error messages radio
+#*************************************************************
+
+error.tax_credits_select = Select yes if you get tax credits
+head.error.tax_credits_select = Select yes if you get tax credits
+error.multiple_address_select = Select yes if you live in more than one place
+head.error.multiple_address_select = Select yes if you live in more than one place
+error.uk_address_select = Select yes if you live in the UK
+head.error.uk_address_select = Select yes if you live in the UK
+error.address_select = Select an address
+head.error.address_select = Select an address
+error.postal_address_uk_select = Select yes if your postal address is in the UK
+head.error.postal_address_uk_select = Select yes if your postal address is in the UK
+
+#*************************************************************
+# Form error date validation
+#*************************************************************
+error.date.required.day = The date you started to live here must include a day
+head.error.date.required.day = The date you started to live here must include a day
```

Comment:
Same again

Reply:
All the views are using the same component for the error summary on top of the page which adds the head. prefix to the error message, if I change it to not use that it will affect other messages and screens that were not being changed in this audit